### DISCUSSION TITLE
How to mount disk for drop volume?

### DISCUSSION THREAD

**Discussion by trueToastedCode** [chris1111/Legacy-Video-patch#25] [Q&A] [answered]
I am trying to install the patch on my system, however how can i drop my disk?

**Comment by chris1111** [comment, marked as answer]
![Capture d’écran, le 2021-02-11 à 10 13 56](https://user-images.githubusercontent.com/6248794/107656270-fe53fd00-6c51-11eb-843b-226e80ae99a3.png)
Finder / Preferances /Chek Box Disk
